packages/languages/src/controllers/LanguageApi.ts

```diff
@@ -15,5 +16,9 @@ export class LanguageApiImpl implements LanguageApi {
      */
     registerLanguage = async (req: Request, res: Response)=> {
         console.log("TODO: 'registerLanguage' not implemented yet " + req + " " + res)
+        lionwebResponse(res, 501, {
```

Comment:
I would consider using a constant for the HTTP code

Reply:
Done, using enums